help/destinations/catalog/advertising/acxiom-prospect-suppression.md

```diff
@@ -0,0 +1,165 @@
+---
+title: Acxiom Prospect-Suppression
+description: Use Acxiom Prospect-Suppression to deliver the most productive prospect audiences possible. This connector securely exports first party data from Real-Time CDP and runs it through an award-winning hygiene and identity resolution which produces a data file to be used as a suppression list. We then match that against the Acxiom Global database which enables the prospect lists to be tailored for import. Acxiom offers the industry’s best-performing audiences with the largest catalog of over 12,000 global data attributes specifically focused on providing personalized experiences. Tap into limitless combinations of high-quality data to create and distribute audiences to meet specific campaign needs.
+last-substantial-update: 2024-01-31
+badge: Beta
+---
+# [!DNL Acxiom Prospect-Suppression] destination connection
+
+>[!NOTE]
+>
+>The [!DNL Acxiom Prospect-Suppression] destination is in beta.  This destination connector and documentation page are created and maintained by the Acxiom team. For any inquiries or update requests, please contact them directly at [11:38 AM] Narisety, K. Deepak (Acxiom)
+acxiom-adobe-help@acxiom.com.
+
+## Overview {#overview}
+
+Use Acxiom Prospect-Suppression to deliver the most productive prospect audiences possible. This connector securely exports first party data from Real-Time Customer Data Platform and runs it through an award-winning hygiene and identity resolution which produces a data file to be used as a suppression list. We then match that against the Acxiom Global database which enables the prospect lists to be tailored for import. Acxiom offers the industry’s best-performing audiences with the largest catalog of over 12,000 global data attributes specifically focused on providing personalized experiences. Tap into limitless combinations of high-quality data to create and distribute audiences to meet specific campaign needs.
+
+This tutorial provides steps to create an [!DNL Acxiom Prospect-Suppression] destination connection and dataflow using the Adobe Experience Platform user interface.  This connector is used to deliver data to Acxiom prospect service using Amazon S3 as a drop point.
```

Comment:
we still haven't clarified what happens to data after the files are dropped into an Amazon S3 location. are the next steps on the customers to ingest those into Acxiom, is there some sort of automated process in place, etc? can we please explain what happens next?

Reply:
Added a Next Steps section after the review section that outlines what a user would need to do to get the batch file this produces linked in with the processing on our side.